posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -813,15 +855,15 @@ public class PostHogReplayIntegration(
     }
 
     private fun View.isNoCapture(maskInput: Boolean): Boolean {
-        return (this.tag as? String)?.lowercase()?.contains("ph-no-capture") == true || maskInput
+        return (tag as? String)?.lowercase()?.contains("ph-no-capture") == true || maskInput
     }
 
-    private fun Drawable.copy(resources: Resources): Drawable? {
-        return constantState?.newDrawable(resources)
+    private fun Drawable.copy(): Drawable? {
+        return constantState?.newDrawable()
     }
 
     private fun String.mask(): String {
-        return "*".repeat(this.length)
+        return "*".repeat(length)
```

Comment:
fly-by & non-blocking should we limit the max length of this?

Reply:
I don't think so otherwise if the label should break into the next line because its too big, it won't because we are truncating the string, so we should have the same amount of chars.